includes/jira.codehaus.org.inc

```diff
@@ -7,8 +7,9 @@
 ##################################
 # Owned by: Olivier Gaudin       #
 ##################################
-RewriteRule "^/browse/SONAR(.*)$" "http://jira.sonarsource.com/browse/SONAR$1" [L]
-RewriteRule "^/browse/MSONAR(.*)$" "http://jira.sonarsource.com/browse/MSONAR$1" [L]
+RewriteRule "^/browse/SONARSCTFS-(.*)$" "http://jira.sonarsource.com/browse/SONARTFVC-$1" [R=301,L]
```

Comment:
Also note that this affects less than 10 tickets in total, so I can even image to hardcode all the mappings in this file